tests/unit/test-util/test.adb.js

```diff
@@ -59,11 +67,13 @@ function getFakeADBKit(
 
   adbUtilReadAllStub.onCall(0).returns(Promise.resolve(Buffer.from('')));
 
+
```

Comment:
Nit, remove this unnecessary empty line (there was already one at line 69 to separate the two statements).